### DISCUSSION TITLE
Hi Everyone!

### DISCUSSION THREAD

**Discussion by karthikraja95** [OpenMined/courses#57] [Introduce Yourself]
This is Karthik, I recently completed my masters from the University of Toronto and I am very excited to start this Private AI series. I hope we will learn a lot of new techniques and methodologies that we can utilize in our day to day life to improve the privacy and security of our data when using machine learning algorithms. Looking forward to working with all of you.

**Comment by iamtrask** [comment]
Hey Karthik! Welcome and congratulations on completing your masters - especially from a place as great as U of T!

**Reply by karthikraja95** [reply]
@iamtrask Thanks! I’m very excited to learn a lot from you!

**Comment by yemikifouly** [comment]
Hi Karthik! 👋 Welcome to the discussion board and to the Private AI Series first course! 🙂 You will definitely learn a lot of new techniques and methodologies that you can utilize in your day to day life to improve the privacy and security or your data. But not only that, you will also learn techniques that can allow access to data that you wouldn't have been able to access because of privacy concerns. 🙂 If you haven't yet, go to the [course's website](https://courses.openmined.org) and start the first course now. 🙂

**Reply by karthikraja95** [reply]
I had a Chance to take a look at the course. I’m going to start the course soon and excited to learn from the experts.
